app/models/grade.rb

```diff
@@ -0,0 +1,9 @@
+class Grade < ApplicationRecord
+  attr_accessor :number
+  start_number = 1
```

Comment:
don't do like this, use constants if you need to set some unchangeable values

Reply:
Wrote down as constants.